testsite/foo/templates/index.html

```diff
@@ -3,19 +3,32 @@
 <html>
 <head>
     <title>Test Civet</title>
-    <link rel="stylesheet" href="{% static 'global.css' %}">
-    <link rel="stylesheet" href="{% static 'shared.css' %}">
-    <link rel="stylesheet" href="{% static 'local.css' %}">
-    <link rel="stylesheet" href="{% static 'library.css' %}">
-    <script src="{% static 'global.js' %}" type="text/javascript" charset="utf-8" async defer></script>
-    <script src="{% static 'shared.js' %}" type="text/javascript" charset="utf-8" async defer></script>
-    <script src="{% static 'local.js' %}" type="text/javascript" charset="utf-8" async defer></script>
-    <script src="{% static 'library.js' %}" type="text/javascript" charset="utf-8" async defer></script>
+    <!-- should be found -->
+    <link rel="stylesheet" href="{% static 'css/global.css' %}">
+    <link rel="stylesheet" href="{% static 'sass/shared.css' %}">
+    <link rel="stylesheet" href="{% static 'css/local.css' %}">
+    <link rel="stylesheet" href="{% static 'sass/library.css' %}">
+
+    <!-- should produce error -->
+    <link rel="stylesheet" href="{% static 'some_managed_package/test.css' %}">
+    <link rel="stylesheet" href="{% static 'bower_components/test/test_styles/main.css' %}">
```

Comment:
`main.css` actually loads after you've run `bower install`, doesn't it?

As for `some_managed_package`, I wonder if you meant `to_be_ignored`, and the purpose is to show that, since the directory is ignored, this is supposed to produce an error if you don't manually compile them, right? I'd explain the intention in the comment here.